DiffPlex.Wpf/Controls/Helper.cs

```diff
@@ -41,15 +41,26 @@ internal static class Helper
     internal static void RenderInlineDiffs(InternalLinesViewer viewer, ICollection<DiffPiece> lines, UIElement source, int contextLineCount)
     {
         viewer.Clear();
+        var diffViewer = source as DiffViewer;
+
         if (lines == null) return;
         if (lines.Any() == false) return;
         var disableSubPieces = lines.Count > MaxCount;    // For performance.
         foreach (var line in lines)
         {
             if (line == null)
             {
-                var c = viewer.Add(null, null, null as string, ChangeType.Unchanged.ToString(), source);
-                c.Tag = line;
+                if (diffViewer.IsTextWrapEnabled)
+                {
+                    var c = viewer.Add(null, null, null as string, ChangeType.Unchanged.ToString(), source);
+                    c.Tag = line;
```

Comment:
This can be done once after the block with a base element for c

Reply:
This has been pushed